src/components/Input/Input.tsx

```diff
@@ -92,7 +114,23 @@ const Input = ({
     }
   }, [takeFocus]);
 
-  return (
+  return type === "checkbox" ? (
```

Comment:
Try to avoid nesting ternaries as it makes it hard to follow.

I think you'll also want to spread the props on these new checkbox and radio inputs as well.

Reply:
Thanks @hatched - refactored. The tests are failing on this `labelFirst` variable, now we have included the checkbox and radio components I'm not sure we need this still - or am I missing something?